notebooks/el-paso/el-paso-no-output.ipynb

```diff
@@ -500,7 +486,10 @@
         "            cases = projected_cases.loc[date, \"New cases\"]\n",
         "            if np.isnan(cases):\n",
         "                raise KeyError\n",
+        "            \n",
         "            # Add some (fairly arbitrary) uncertainty around this point estimate\n",
+        "            if cases == 0:\n",
```

Comment:
this had to be added because we now sometimes project 0 cases (we didn't before)